ctapipe/reco/tests/test_sklearn.py

```diff
@@ -58,6 +58,7 @@ def example_table():
         n_samples=100, n_features=5, n_informative=3, random_state=0
     )
     t = Table({f"X{i}": col for i, col in enumerate(X.T)})
+    t["tel_id"] = np.arange(len(t))
```

Comment:
Maybe it is better to not groupby tel id if it is not there instead of relying on it being attached? Still would have the default value of the TelescopeParameter